blocks/extra-large-promo-block/_index.scss

```diff
@@ -1,6 +1,10 @@
 @use "@wpmedia/arc-themes-components/scss";
 
 .b-xl-promo {
+	&__img {
+		@include scss.block-properties("xl-manual-promo-img");
+		@include scss.block-components("xl-manual-promo-img");
```

Comment:
These should be `xl-promo-img`.

Reply:
@vgalatro Thanks, fixed it [808d50b](https://github.com/WPMedia/arc-themes-blocks/pull/1690/commits/808d50ba8aceca28b1da67cb8472f5949c731f92)